TypeCobol/Tools/Options-Config/TypeCobolConfiguration.cs

```diff
@@ -56,6 +59,14 @@ public class TypeCobolConfiguration : ITypeCobolCheckOptions
         public string RawMaximumDiagnostics;
         public string RawOutputFormat = "0";
 
+#if EUROINFO_RULES
+        public static string DefaultCopyNameFile => Path.Combine(Path.GetDirectoryName(System.Reflection.Assembly.GetEntryAssembly().Location),"COPIES_CPY.txt");
```

Comment:
Actually I don't think we need a default value for this. This is an EI-specific option so it will be used in a controlled environment and we can ensure that it will be correctly set. For public users, a default value would not give any benefit.
@rooksdo, you wanted a default value, what use case was on your mind ?

Reply:
done